unable to run mfe app on windows: module not found MFClient.js

Noticed something when trying to run the mfe app on windows. On mac, I had no issues starting up my apps. Was wondering if there is a solution for this. Checking the modules folder, I noticed we have a [`patchNextClientPageLoader.js`](https://github.com/module-federation/nextjs-mf/blob/main/packages/nextjs-mf/src/loaders/patchNextClientPageLoader.ts) file that was meant to resolve this issue however it doesn't seem to be working.

```
customer: error - ../../node_modules/next/dist/client/page-loader.js:111:23
customer: Module not found: Can't resolve '..\..\..\@module-federation\nextjs-mf\client\MFClient.js'
customer: Did you mean './..\..\..\@module-federation\nextjs-mf\client\MFClient.js'?
customer: Requests that should resolve in the current directory need to start with './'.
customer: Requests that start with a name are treated as module requests and resolve within module directories (node_modules, C:\Users\Raymond\next-nx-mfe).
customer: If changing the source code is not an option there is also a resolve options called 'preferRelative' which tries to resolve these kind of requests in the current directory too.
customer: Import trace for requested module:
customer: ../../node_modules/next/dist/client/index.js
customer: ../../node_modules/next/dist/client/next-dev.js
customer: https://nextjs.org/docs/messages/module-not-found
```

**my dependencies:**
@module-federation/nextjs-mf: "^5.11.0"
next: 12.3.1

--- a/packages/nextjs-mf/src/loaders/async-boundary-loader.ts
+++ b/packages/nextjs-mf/src/loaders/async-boundary-loader.ts
@@ -91,7 +91,7 @@ export const pitch = function( this: LoaderContext<Record<string, unknown>>, rem
         );
 
         const result = [
-          pageTemplate(`${this.resource}?hasBoundary`)
+          pageTemplate(`${relativeResource}?hasBoundary`)
         ];
 
         if(hasGIP) {

